components/FireImpactCharts.vue

```diff
@@ -88,7 +88,8 @@ const speciesOptions = [
 
 const renderPlot = () => {
   let traces = []
-  let fmoData = store.fireImpactData[fmoSelection.value][speciesSelection.value]
+  let fmoData =
+    store.areaData['fireImpact'][fmoSelection.value][speciesSelection.value]
```

Comment:
I think I can answer my own question after digging deeper.  It looks like if a file is missing or malformed, this could happen but I don't know whether it's worth coding defensively around that.  🤷

Reply:
Good point, I think it is worth going through the code more carefully to see where more defensive coding can be added. I'll create an issue for this because there are surely other places that could benefit from a more defensive approach.

Reply:
Captured in #83.